### DISCUSSION TITLE
[PSD5] Exercise 2.3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#140] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210014973-45e55315-85bc-47fb-85c4-6d766a22d6dd.png)

- _**N.B.:** Le tavole della verità sono le stesse dell'Esercizio 2.1 (#138 ), ma questa volta l'equazione booleana va scritta in forma canonica product-of-sums (POS)._

**Comment by Elia-Belli** [comment]
**Forma Canonica POS**: non minimizzata
(a) $\prod{(1)} = A + \bar B$
(b) $\prod{(1,2,3,4,5,6)}= (A + B + \bar C)(A + \bar B + C)( A + \bar B + \bar C)(\bar A + B + C)(\bar A + B + \bar C)(\bar A + \bar B + C)$
(c) $\prod{(1,3,6)}= (A + B + \bar C)(A + \bar B + \bar C)(\bar A + \bar B + C)$
(d) $\prod{(4,5,6,7,9,11,12,13,15)}= (A + \bar B +C + D)(A + \bar B + C + \bar D)( A + \bar B + \bar C + D)(A + \bar B + \bar C+ \bar D)$
$(\bar A + B + C + \bar D)(\bar A + B + \bar C + \bar D)(\bar A + \bar B + C + D)(\bar A + \bar B + C + \bar D)(\bar A + \bar B + \bar C+ \bar D)$
(e) $\prod{(1,2,4,7,8,11,13,14)}= (A + B +C + \bar D)( A +  B + \bar C + D)(A + \bar B + C + D)(A + \bar B + \bar C + \bar D)(\bar A +  B + C + D)$
$(\bar A + B + \bar C + \bar D)(\bar A + \bar B + C + \bar D)(\bar A + \bar B + \bar C + D)$

**Comment by FeddyLix17** [comment]
@Elia-Belli 
(d) tutto corretto manca solo m<sub>15</sub>, che è (A&#x0304;B&#x0304;C&#x0304;D&#x0304;)
per il resto tutto corretto

**Comment by Elia-Belli** [comment]
@FedVlogger17 corretto, penso intendessi $(\bar A + \bar B + \bar C+ \bar D)$

**Comment by FeddyLix17** [comment]
@Elia-Belli ops, si esatto intendevo quello grazie per avermelo fatto notare
